src/Menu.php

```diff
@@ -54,11 +54,13 @@ public function __construct($title = '', array $options = [])
      * @param mixed $value
      * @param string $label
      *
+     * @param CliMenuBuilder|null $subMenu
      * @return \NunoMaduro\LaravelConsoleMenu\Menu
      */
-    public function addOption($value, string $label): Menu
+    public function addOption($value, string $label, ?CliMenuBuilder $subMenu = null): Menu
```

Comment:
Hey buddy, why we need the `, ?CliMenuBuilder $subMenu = null`?

Reply:
You are 100% right, we don't. It was a left-over from a previous implementation I tried. I removed it now.

Reply:
In the callback on line 92, I pass the Menu object so we can add options to the menu recursively.

Reply:
I assigned the values to new variables to make it a bit clearer what's happening.

Reply:
@nunomaduro @owenvoke any further thoughts on this? Trying to get a decision on this before I release my book (June 1st) so I can scratch it or leave it in or change it.

Reply:
Hey, sorry I seem to have a huge amount of notifications recently so think I missed this again. 🤦 Personally I think this seems fine.

Reply:
Any new thoughts on this @nunomaduro ?

Reply:
@nunomaduro fixed the merge conflict, good to go!

Reply:
@nunomaduro i think your styleci might be misconfigured, it wants double spaces here;

![image](https://user-images.githubusercontent.com/431360/202894868-3512b717-8ae7-4430-8cf4-20bcfba29735.png)

Anyway, passes now. Weird.

Reply:
Double spaces is the Laravel standard. Tbh, we should probably migrate to Pint here.